packages/terra-table/src/subcomponents/SubSection.jsx

```diff
@@ -0,0 +1,90 @@
+// TODO: fix linter error
```

Comment:
Is this TODO addressed, or is it out of scope of this PR?

Reply:
Looks like it was from a previous PR but there doesn't seem to be any linter issues. Removed here: 86bd928341181f2152cd32ac5db908084fce26b1